compiler_gym/util/GrpcStatusMacros.h

```diff
@@ -4,10 +4,18 @@
 // LICENSE file in the root directory of this source tree.
 #pragma once
 
+#include "glog/logging.h"
 #include "include/grpcpp/grpcpp.h"
 
 using grpc::Status;
 
+#undef ASSERT_OK
+#define ASSERT_OK(expr)                             \
```

Comment:
Do you need the `do .. while()`? Won't just a block do?

Reply:
It's what all the cool kids do. I had no idea why, until I googled it. Still not sure I understand :) https://stackoverflow.com/a/4674580